Prac 07 feedback

Ch4insawPanda opened the pull request:
@Herophillix Please help me check function/argument names and format of the code.

Ch4insawPanda committed:
-Add new buttons 'Clear' and 'Greet'
-Add new text input field
-Create event handlers to display name and clear fields

Ch4insawPanda committed:
-reformat display to show horizontally
-add instructional label at the bottom

Ch4insawPanda committed:
Create a widget that converts miles to km

Ch4insawPanda committed:
Create a widget that creates labels from a list

Herophillix commented:
Looks good overall, some minor issues

Consider changing `int()` into `float()` as user input can be a `float`
https://github.com/Ch4insawPanda/CP1404_Practical/blob/d7e39361c1617ae943350c42519980135e6d814e/prac_07/convert_miles_km/convert_miles_km.py#L23-L25

Consider commenting it out as requested in the practical
https://github.com/Ch4insawPanda/CP1404_Practical/blob/d7e39361c1617ae943350c42519980135e6d814e/prac_07/convert_miles_km/convert_miles_km.kv#L17-L20

Consider adding a check before running the application similar to 
```
if __name__ == "__main__":
    main()
```
https://github.com/Ch4insawPanda/CP1404_Practical/blob/d7e39361c1617ae943350c42519980135e6d814e/prac_07/convert_miles_km/convert_miles_km.py#L41
https://github.com/Ch4insawPanda/CP1404_Practical/blob/d7e39361c1617ae943350c42519980135e6d814e/prac_07/dynamic_labels/dynamic_labels.py#L27
https://github.com/Ch4insawPanda/CP1404_Practical/blob/d7e39361c1617ae943350c42519980135e6d814e/prac_07/squaring/squaring.py#L31
https://github.com/Ch4insawPanda/CP1404_Practical/blob/d7e39361c1617ae943350c42519980135e6d814e/prac_07/box_layout_demo.py#L19

Herophillix commented:
Consider changing these two to using `StringProperty()`
https://github.com/Ch4insawPanda/CP1404_Practical/blob/d7e39361c1617ae943350c42519980135e6d814e/prac_07/box_layout.kv#L13-L16
https://github.com/Ch4insawPanda/CP1404_Practical/blob/d7e39361c1617ae943350c42519980135e6d814e/prac_07/box_layout.kv#L7-L9

Inconsistent texts between original and new text
https://github.com/Ch4insawPanda/CP1404_Practical/blob/d7e39361c1617ae943350c42519980135e6d814e/prac_07/box_layout.kv#L13-L16
https://github.com/Ch4insawPanda/CP1404_Practical/blob/d7e39361c1617ae943350c42519980135e6d814e/prac_07/box_layout_demo.py#L15

Ch4insawPanda committed:
-Rename get_value to validate_value
-Change int to float when coverting from str
-Comment out convert button as stated in practical

Ch4insawPanda committed:
-Rename get_value to validate_value
-Change int to float when coverting from str
-Comment out convert button as stated in practical

Ch4insawPanda committed:
-Rename get_value to validate_value
-Change int to float when coverting from str
-Comment out convert button as stated in practical

Ch4insawPanda committed:
-Fix inconsistent text
-Add string property to output_label
